lib/spoom/cli/commands/bump.rb

```diff
@@ -0,0 +1,85 @@
+# typed: false
+# frozen_string_literal: true
+
+require 'find'
+require 'open3'
+
+require_relative 'base'
+
+module Spoom
+  module Cli
+    module Commands
+      class Bump < Base
+        extend T::Sig
+
+        default_task :bump
+
+        desc "bump", "bump Sorbet sigils from `false` to `true` when no errors"
+        sig { params(directory: String, extension: String).returns(T::Array[String]) }
+        def bump(directory = ".", extension = "rb")
+          files_to_bump = Bump.files_with_sigil_strictness(directory, "false", extension)
```

Comment:
Why not allowing the user to specify the strictness?


```
spoom bump --from ignore --to false
```

Reply:
Done, thanks.